DOCUMENTATION.md

```diff
@@ -615,6 +616,87 @@ var page = PO.build({
 
 Note that if the plain object doesn't have attributes defined, the object is returned as is.
 
+### `.customHelper`
+
+Defines a way in which custom components can be created using
+information of their surrounding context.
+
+It expects a function with the following signature:
+
+```js
+function(selector, options) {
+  // user magic goes here
+  return Value;
+}
+```
+
+`selector` includes all the knowledge of the context in which
+the function will be called (you can see examples of usage below).
+
+To understand which should the return value for this function, please
+check the code examples below.
+
+```js
+var input = customHelper(function(selector, options) {
+  return {
+    value: value(selector, options)
+  };
+});
+
+var page = PageObject.build({
+  search: input('.header input'),
+
+  users: collection({
+    itemScope: 'tr',
+    item: {
+      userName: input('input:first')
+    }
+  })
+});
+```
+
+If we execute:
+
+`page.users(3).userName()`
+
+It would look for `tr:nth-of-type(3) input:first` in our page.
+
+In this example, the user defined function returns a POJO which
+will be converted into a `PageObject` component.
+
+Here is another two examples of using `customHelper` but this time, the
+user defined functions return a function and a string.
+
+```js
+var admin = PO.customHelper(function(selector, options) {
+  return function() {
+    return $(selector).hasClass('admin');
+  };
+});
+
+var disabled = PO.customHelper(function(selector, options) {
+  return $(selector).prop('disabled');
+});
+
+var page = PO.build({
+  myText: {
+    scope: '#myInput',
+    isAdminField: admin(),
+    isDisabled: disabled()
```

Comment:
Can we change `disabled` and `admin` function names to be `isDisabled` and `isAdmin`? So we follow the same convention as the rest of the project